app/convbin/gcc/makefile

```diff
@@ -158,5 +158,12 @@ test4:
 test5: test1 test4  # make sure JSON with no spaces are bit exact minus first two lines
 	tail -n +5 ./2017-05-12-v1.1.26-json-nospaces.obs > 2017-05-12-v1.1.26-json-nospaces.trunc.obs
 	diff 2017-05-12-v1.1.26-json-nospaces.trunc.obs 2017-05-12-v1.1.26-bin.trunc.obs
+test6: # BDS + GAL + GLO + GPS
+	@echo -e "\n\n****** Test6: BDS + GAL + GLO + GPS ********\n"
+	./sbp2rinex $(DATDIR)$(SEP)2018-06-15-v1.4.0-develop-2018061321-3-g734bd4b9.sbp.json -o 2018-06-15-json.obs -n 2018-06-15-json.nav
+	tail -n +3 ./2018-06-15-json.obs > 2018-06-15-json.trunc.obs
```

Comment:
So you are comparing only the last 3 lines of the file? Did you try comparing the entire file (apart from the first header line?

Reply:
As far as I understand, this checks all starting from 3rd line:

```
   -n, --lines=K
       Output the last K lines, instead of the last 10; or use -n +K to output lines starting with the Kth 
```

And it's pretty much a copy-paste from other cases.

Reply:
Sounds good, sorry, I mis-read the +